### PR TITLE
vote: JmPotato as tikv reviewer

### PR DESCRIPTION
**Pull request by Connor1996**
Signed-off-by: Connor1996 <zbk602423539@gmail.com>

### CODE REVIEW

**Review by tonyxuqqi — APPROVED** [review]

**Comment by JmPotato** [comment]
cc @tikv/maintainers 

Please take a look, thx a lot.

**Comment by JmPotato** [comment]
cc @BusyJay @hicqu @breezewish @innerr 

Please take a look, thx a lot.

**Review by zhangjinpeng1987 — APPROVED** [review]
+1 LGTM

File: votes/0188-JmPotato-as-tikv-reviewer.md
Comment (JmPotato) [inline votes/0188-JmPotato-as-tikv-reviewer.md:17]: Maybe we should update the result before merging.

**Review by JmPotato — CHANGES_REQUESTED** [review]

**Comment by BusyJay** [comment]
As said in the poll,

> The vote will be open for at least 3 days unless there is an objection or not enough votes.

Don't hurry to merge.

File: votes/0188-JmPotato-as-tikv-reviewer.md
Comment (JmPotato) [reply votes/0188-JmPotato-as-tikv-reviewer.md:17]: @Connor1996 Hello, could we update the vote result and merge this PR now?

**Comment by skyzh** [comment]
I'll update the result

**Pull request merged**
